index.js

```diff
@@ -102,3 +100,50 @@ module.exports.isMACValid = function(mac) {
 
     return !(mac.length != 2 * MAC_BYTES || mac.match(/[^a-fA-F0-9]/));
 }
+
+module.exports.relay = function(mac, params) {
+
+        getIP(params).then(function(ip) {
+            
+            var magicPacket = createMagicPacket(mac);
+            listensocket = dgram.createSocket('udp4');
+            
+            listensocket.on('error', (err) => {
+              console.log(`server error:\n${err.stack}`);
+              listensocket.close();
+            });
+            
+            listensocket.on('message', (msg, rinfo) => {
+            
+              if (msg.equals(magicPacket))
+              {
+                var broadcastsocket = dgram.createSocket(net.isIPv6(ip) ? 'udp6' : 'udp4');
+                
+                broadcastsocket.once('listening', function() {
```

Comment:
why you don't use arrow functions here? :)

Reply:
Just looked them up and read about them!  I'll make the change.